packages/components/src/color-palette/utils.ts

```diff
@@ -92,9 +92,13 @@ export const normalizeColorValue = (
 	value: string | undefined,
 	element: HTMLElement | null
 ) => {
+	if ( ! value || ! element ) {
```

Comment:
There was a related conversation about how to handle null values here: https://github.com/WordPress/gutenberg/pull/64224#discussion_r1721472283

Not sure if I've grokked the outcome of that, so let me know if there's a better way to do this.